cafebabel/static/js/script.js

```diff
@@ -51,21 +51,51 @@ if (socialIcons) {
     })
 }
 
+function activateInput(input) {
+  const parent = input.parentElement
+  parent.classList.add('active', 'completed')
+}
+function deactivateInput(input) {
+  const parent = input.parentElement
+  parent.classList.remove('active', 'completed')
+}
+
+/* Detect chrome autofill https://stackoverflow.com/questions/35049555/chrome-autofill-autocomplete-no-value-for-password/38269071 comm ben */
```

Comment:
The actual permalink to Ben's comment is https://stackoverflow.com/a/40852860 (there's a _share_ link on every comment that links on each)

Reply:
didn't know ;-)